css/header.css

```diff
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  max-width: 100vw;
+}
+
+.logo {
+  font-family: "Libre Baskerville", serif;
+}
+
+.nav-head {
+  position: absolute;
+  min-width: 100%;
+  top: 0;
+  background-color: var(--primary-color-bg);
+  padding: var(--spacing-xs);
+  font-family: "Nunito", sans-serif;
```

Comment:
try to keep fonts liimited to 2 or 3 at most. font load takes time and bandwidth, check lighthouse